packages/relay-compiler/codegen/writeRelayGeneratedFile.js

```diff
@@ -111,11 +118,15 @@ async function writeRelayGeneratedFile(
             text != null,
             'writeRelayGeneratedFile: Expected `text` in order to persist query',
           );
+          invariant(
+            persistedQueryId != null,
+            'writeRelayGeneratedFile: Expected `persistedQueryId` in order to persist query',
```

Comment:
Unfortunately this was required for Flow but I can't see how it would possibly be hit with the current logic.